src/main/webapp/js/datahandlers/utils.js

```diff
@@ -19,11 +19,42 @@ goog.module('datahandlers.utils');
 /** 
  * Converts camelcase string to phrases.
  * @param {string} str The camelcase string.
- * @returns The formatted string.
+ * @returns {string} The formatted string.
  */
 const addSpaceToCamelCase = (str) => {
   let result = str.replace( /([A-Z])/g, " $1" );
   return result.charAt(0).toUpperCase() + result.slice(1);
-}
+};
 
-exports = {addSpaceToCamelCase};
\ No newline at end of file
+/**
+ * Converts decimal to a string representing a percent.
+ * @param {number} num - The decimal to convert.
+ * @returns {string} - The string representation of the number as a percent.
+ */
+const convertToPercent = (num) => {
+  num *= 100;
+  return round_(num, 2).concat(' %');
+};
```

Comment:
Next time you can use `number..toFixed(/** precision= */2)`.

Reply:
Good idea! Will change